gitpod-io/website
# [Docs] Clarify how onOpen behaves for JB

## Description
Update `.gitpod.yml` reference page, specifically the `onOpen` section. For JetBrains IDEs, once we deploy https://github.com/gitpod-io/gitpod/pull/10107 we will have a similar behaviour to the VS Code experience in terms of notifications and behavior for `onOpen`. However, there is a tiny difference which this PR aims to highlight.

<img width="807" alt="Screenshot 2022-05-30 at 17 43 16" src="https://user-images.githubusercontent.com/2318450/171065453-82ef1aa9-db03-4f2e-8f83-943b43e2927f.png">





## Related Issue(s)

Fixes #2058 

## How to test
Check on https://deploy-preview-2146--gitpod-kumquat.netlify.app/docs/references/gitpod-yml#portsnonopen

## Release Notes

```release-note
```

## Documentation



<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2146"><img src="https://gitpod-staging.com/button/open-in-gitpod.svg"/></a>




<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2146"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
